.github/workflows/run_backend_services_tests.yml

```diff
@@ -13,18 +13,26 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [18.x]
-        service: [user-service, question-service, collaboration-service, history-service, matching-service]
+        node-version: [16.x, 18.x, 20.x]
```

Comment:
Is there a reason why we need 16.x? I believe the `fetch()` api does not exist on node 16 so users can't use our app without using a node version of 18 and higher

Reply:
Ah the reason why I put NodeJS v16.x is to eliminate the possibility that the "graders" who have old node version can still run our code. 

>  I believe the fetch() api does not exist on node 16

I see, then I will just remove node v16 :)